src/nn/NeuralNetworkContext.js

```diff
@@ -11,7 +11,7 @@ export default class NeuralNetworkContext {
     this._initFusedActivationFunctionTypes();
     this._initImplicitPaddingTypes();
     this._initExecutionPreferenceTypes();
-    this.supportWebGL = WebGLModel._supportWebGL();
+    this.supportWebGL = TfjsModel._supportWebGL();
```

Comment:
This line does the same thing as await [tf.setBackend('webgl'); ](https://github.com/intel/webml-polyfill/pull/1153/files?file-filters%5B%5D=.js#diff-e65d05ca7955d54cba88664b63745133R43), it would be better to keep just one of them.

Reply:
Yes, so I use `tf.setBackend('webgl')` only if backend is 'webgl' but tfjs backend isn't 'webgl'.

Reply:
@NALLEIN , are you fine with this?